packages/cli/commands/export/credentials.ts

```diff
@@ -98,8 +98,8 @@ export class ExportCredentialsCommand extends Command {
 				);
 				logger.error('\nFILESYSTEM ERROR');
 				logger.info('====================================');
-				logger.error(e.message);
-				logger.error(e.stack);
+				logger.error((e as Error).message);
+				logger.error((e as Error).stack ?? '');
```

Comment:
I wonder if it'd be cleaner to use `if (error instanceof Error) {...}` or as a guard `if (!(error instanceof Error)) return;`

Reply:
I was trying to avoid modifying logic in places I couldn't quickly test due to the amount of changes required.

Reply:
My reasoning is that we are already assuming we can access error-specific properties all over, so if the value being force-asserted `as Error` is not actually an error, the logic is bound to break or resolve to `undefined`, so `if (error instanceof Error) {...}` would only change logic to the extent that it would prevent those outcomes. But please correct me if I'm wrong.

Reply:
> a lot of places use parts of the errors then throw another more specific error up

Can you link to an example of this to understand better? 

In my mind the goal of enabling this flag is to increase type safety by requiring us to refine the error type before usage in order to prevent us from relying on non-existing properties, and this is what type guards help with e.g. `if (error instanceof SomeCustomError)`, but maybe I'm not fully seeing the issue you mean.

Reply:
Independent of the rest of the conversation here, I'd like to point out that 
1. `(e as Error).message` has no advantage over `e.message` (when `e` is `any`). `instanceof` operator on the other hand adds a real check.
2. `no-throw-literal` helps us avoid strings being thrown in the code, but it does not help with any thrown strings coming from one of the hundreds of packages that we use.